aptos/typescript/backend_examples/src/invisible_wallet.ts

```diff
@@ -0,0 +1,101 @@
+// 1. Import everything we need for the tutorial
+import { 
+    AccountAddress,
+    AccountAuthenticatorEd25519,
+    Aptos, 
+    AptosConfig, 
+    Network, 
+    SimpleTransaction 
+} from "@aptos-labs/ts-sdk";
+import { 
+    KeyClient, 
+    WalletClient, 
+    ShinamiWalletSigner  
+} from "@shinami/clients/aptos";
+
+// 2. Copy your access key value
+const ALL_SERVICES_TESTNET_ACCESS_KEY = "{{allServicesTestnetAccessKey}}";
+
+// 3. Set up a walletId and its associated secret. Just for the tutorial. Your
+//    app should figure out the best way to manage its wallet IDs and secrets.
+const WALLET_ID = "{{walletID}}";
+const WALLET_SECRET = "{{walletSecret}}";
+
+// 4. Instantiate your Aptos and Shinami clients
+const aptosClient = new Aptos(new AptosConfig({ network: Network.TESTNET }));
+const keyClient = new KeyClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+const walletClient = new WalletClient(ALL_SERVICES_TESTNET_ACCESS_KEY);
+
+// 5. Create a ShinamiWalletSinger to more easily manage the Invisible Wallet
+const signer = new ShinamiWalletSigner(
+    WALLET_ID,
+    walletClient,
+    WALLET_SECRET,
+    keyClient
+);
+
+// 6. Create the Invisible Wallet. The call to `executeGaslessTransaction` below
+//    would initailize an un-initialized wallet. However, we are explicitly
+//    initializing it now in case you only run the `signAndVerifyTransaction` 
+//    function, because a wallet must be initialized in order to sign a transaction.
+const CREATE_WALLET_IF_NOT_FOUND = true;
+const INITIALIZE_ON_CHAIN = true; 
+const walletAddress = await signer.getAddress(CREATE_WALLET_IF_NOT_FOUND, INITIALIZE_ON_CHAIN);
+
+// 7. Sponsor and execute a transaction for the Invisible Wallet. This requires
+//     the Gas Station fund associated with this access key to have sufficient 
```

Comment:
I would move or mention the gas station fund association in step 6 as well, since we are initializing on chain which also requires gas funds